app/models/miq_report/search.rb

```diff
@@ -93,11 +93,11 @@ def paged_view_search(options = {})
     search_options = options.merge(:class => db, :conditions => conditions, :include_for_find => includes)
     search_options.merge!(:limit => limit, :offset => offset, :order => order) if order
 
-    if options[:parent]
-      targets = get_parent_targets(options[:parent], options[:association] || options[:parent_method])
-    else
-      targets = db_class
-    end
+    targets = if options[:parent] && options[:parent][:class] != "ManageIQ::Providers::Amazon::CloudManager"
```

Comment:
@chessbyte thank you, you are right, and also this change is for now irrelevant because the code has changed and the fix does not work anymore. so I have added WIP to this PR until I find another way to fix the bug.